Packages/com.unity.inputsystem/InputSystem/Devices/Commands/SetIMECursorPositionCommand.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Runtime.InteropServices;
+using UnityEngine.Experimental.Input.Utilities;
+
+namespace UnityEngine.Experimental.Input.LowLevel
+{
+    /// <summary>
+    /// Sets the position for IME dialogs.  This is in pixels, from the upper left corner going down and to the right.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit, Size = InputDeviceCommand.kBaseCommandSize + (sizeof(float) * 2))]
+    public unsafe struct SetIMECursorPositionCommand : IInputDeviceCommandInfo
+    {
+        public static FourCC Type { get { return new FourCC('I', 'M', 'E', 'P'); } }
+
+        [FieldOffset(0)]
+        public InputDeviceCommand baseCommand;
+
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize)]
+        float xPosition;
+
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + sizeof(float))]
+        float yPosition;
+
+        public FourCC GetTypeStatic()
+        {
+            return Type;
+        }
+
+        public static SetIMECursorPositionCommand Create(Vector2 position)
+        {
+            return new SetIMECursorPositionCommand
+            {
+                baseCommand = new InputDeviceCommand(Type, InputDeviceCommand.kBaseCommandSize + (sizeof(float) * 2)),
```

Comment:
Used a kSize constant like other files, but I can't use sizeof(Vector2), because I can't use it for the [StructLayout(LayoutKind.Explicit, Size = kSize)] at the top.  I don't want to have 2 different ways of calculating size.